utils/select.ts

```diff
@@ -29,24 +29,34 @@ const createIndicatorOptionsFromTheme = (theme: SpotlightTheme): SelectOptions =
 };
 
 // TODO: exclude specified years
-export const createYearOptionsFromRange = (startYear = 0, endYear = 0, _exclude?: number[]): SelectOptions => {
+export const createYearOptionsFromRange = (
+  startYear = 0,
+  endYear = 0,
+  excludedYears: string | undefined
```

Comment:
This is long ... optional is `excludedYears?: string`

Reply:
Okay